newIDE/app/src/ExpressionAutocompletion/index.js

```diff
@@ -78,7 +79,7 @@ export type ExpressionAutocompletion =
 type ExpressionAutocompletionContext = {|
   gd: libGDevelop,
   project: gdProject,
-  projectScopedContainers: gdProjectScopedContainers,
+  projectScopedContainers: ProjectScopedContainers,
```

Comment:
Here and everywhere where it's used (see my other comment):
```suggestion
  projectScopedContainersAccessor: ProjectScopedContainersAccessor,
```